StripeFinancialConnections/StripeFinancialConnections/Source/Web/ContinueStateView.swift

```diff
@@ -9,75 +9,82 @@ import UIKit
 @_spi(STP) import StripeCore
 @_spi(STP) import StripeUICore
 
+@available(iOSApplicationExtension, unavailable)
 class ContinueStateView: UIView {
```

Comment:
IF (not sure if that's true) prepane looks exactly the same as this `ContinueStateView` it may be worth considering combining them.

If I am correct in understanding, it would be as simple as just adding an enum and changing the strings if the enum is different (everything else remains the same?)

That being said, 'copying' code sometimes is better, so not necessarily suggesting that this is definitely the right call. Right now there is some "forced" code sharing where Prepane lives in one places, ContinueState lives in another place, and `CreatePartnerDisclosureView` function lives awkwardly in-between.

Reply:
yeah totally. I thought about it, but then didn't want to make the Prepane even more complext, but I does kinda make senes to use it, Initially I was planning to use even less but ended up using most of it. So combining makes more sense.